allennlp/modules/elmo.py

```diff
@@ -499,14 +532,31 @@ def forward(self,  # pylint: disable=arguments-differ
         Note that the output tensors all include additional special begin and end of sequence
         markers.
         """
-        token_embedding = self._token_embedder(inputs)
-        type_representation = token_embedding['token_embedding']
-        mask = token_embedding['mask']
+        if self._word_embedding is not None:
+            mask_without_bos_eos = (inputs > 0).long()
+            # The character cnn part is cached - just look it up.
+            embedded_inputs = self._word_embedding(inputs) # type: ignore
```

Comment:
Ah, so mypy _isn't_ happy with it =).  A type annotation on the initial assignment would fix this, right?